CMakeWorkflowPresets.json

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
What's the difference with CMakePresets?

Reply:
It is like your task.
It makes all steps in a predefined order: configure, build, test, install, ...